hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -192,7 +192,7 @@ public boolean isRunning() {
   @SuppressFBWarnings(value="NN_NAKED_NOTIFY",
       justification="Used only for testing")
   public synchronized void processContainersNow() {
-    notify();
+    notifyAll();
```

Comment:
Here replication manager is a single thread, I think notify should be fine here. Replacing with notifyAll should not cause any harm here.